installer/bundle/create_bundle.sh

```diff
@@ -155,8 +159,13 @@ OUTPUT_DIR=`(cd $2; pwd -P)`
 
 cd $INTERMEDIATE_DIR
 
+if [ ! -f $SOURCE_DIR/bundle_skel_${PLATFORM_TYPE}.sh ]
+then
+    echo "shell bundle skeleton file bundle_skel_${PLATFORM_TYPE}.sh does not exist in ${SOURCE_DIR}"
+    exit 1
+fi
 # Fetch the bundle skeleton file
-cp $SOURCE_DIR/bundle_skel.sh .
+cp $SOURCE_DIR/bundle_skel_${PLATFORM_TYPE}.sh ./bundle_skel.sh
```

Comment:
Whoops, somehow I missed that. My mistake, you're totally correct. Sorry 'bout that.